### ISSUE TITLE
Regression from 3.2.12 to 3.2.13: can't setup ioBroker from scratch

### ISSUE THREAD

**Issue by UncleSamSwiss** [ioBroker/ioBroker.js-controller#1229]
If I try to install js-controller 3.2.13 (under Windows 10) in an "empty" dierctory with an existing `iobroker-data\iobroker.json`, some commands fail. This works fine with 3.2.12.

Procedure:
1. Create `package.json`
```json{
  "name": "test",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "iobroker.js-controller": "3.2.13",
    "iobroker.admin": "latest",
    "iobroker.info": "latest"
  }
}
```
2. Create `iobroker-data\iobroker.json`
```json
 {
  "system": {
    "memoryLimitMB": 0,
    "instanceStartInterval": 2000,
    "compact": false,
    "allowShellCommands": false,
    "memLimitWarn": 100,
    "memLimitError": 50
  },
  "multihostService": {
    "enabled": false
  },
  "network": {
    "IPv4": true,
    "IPv6": false,
    "bindAddress": "127.0.0.1",
    "useSystemNpm": true
  },
  "objects": {
    "type": "file",
    "host": "127.0.0.1",
    "port": 19901,
    "noFileCache": false,
    "maxQueue": 1000,
    "connectTimeout": 2000,
    "writeFileInterval": 5000,
    "dataDir": "",
    "options": {
      "auth_pass": null,
      "retry_max_delay": 5000,
      "retry_max_count": 19,
      "db": 0,
      "family": 0
    }
  },
  "states": {
    "type": "file",
    "host": "127.0.0.1",
    "port": 19900,
    "connectTimeout": 2000,
    "writeFileInterval": 30000,
    "dataDir": "",
    "options": {
      "auth_pass": null,
      "retry_max_delay": 5000,
      "retry_max_count": 19,
      "db": 0,
      "family": 0
    }
  },
  "log": {
    "level": "debug",
    "maxDays": 7,
    "noStdout": false,
    "transport": {
      "file1": {
        "type": "file",
        "enabled": true,
        "filename": "log/iobroker",
        "fileext": ".log",
        "maxsize": null,
        "maxFiles": null
      }
    }
  },
  "plugins": {},
  "dataDir": "../../iobroker-data/"
}
```
3. Call `npm install --loglevel error --production`
4. Now call `node node_modules/iobroker.js-controller/iobroker.js upload admin`
This will show the warning "Cannot read system.config: null (OK when migrating or restoring)"
This warning is not present in 3.2.12
5. Call `node node_modules/iobroker.js-controller/iobroker.js add admin 0`
This will fail with the following error:
```text
 Cannot find view "system" for search "instance"
Error: Cannot find view "system"
    at ObjectsInMemoryServerClass._getObjectView (....\node_modules\@iobroker\db-objects-redis\lib\objects\objectsInRedisClient.js:3292:59)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

It is normal to see the warning "Cannot read system.config: null (OK when migrating or restoring)" when installing js-controller (during "setup first") but it shouldn't show up as a warning when executing other commands (as the system is now set up).

I know, this is an edge case, but I'm worried the problem exists in more real-world scenarios.

**Comment by Apollon77** [comment]
In a "complete fresh" system the first command you need to execute is "iobroker setup first" ... or this executed in step #3?

**Comment by Apollon77** [comment]
PS: Could it be that you updated npm? Can you really reporo that by installing iobroker.js-controller@3.2.12 vs @3.2.13?

**Comment by Apollon77** [comment]
I was able to repro ... try with 3.2.14 please ...

**Comment by UncleSamSwiss** [comment]
I can confirm, this was fixed with 3.2.14.
